adapters/ttx/ttx.go

```diff
@@ -0,0 +1,76 @@
+package ttx
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"net/http"
+)
+
+type TtxAdapter struct {
+	endpoint string
+}
+
+func (a *TtxAdapter) MakeRequests(request *openrtb.BidRequest) ([]*adapters.RequestData, []error) {
+	var errs []error
+	jsonData, err := json.Marshal(request)
+
+	if err != nil {
+		errs = append(errs, err)
+	}
```

Comment:
Do we really need to continue processing the request if an error occurred at this point?  That is: it seems like the code should just return `nil, errs` at this point.